backends/platform/ios7/ios7_osys_main.cpp

```diff
@@ -112,6 +111,37 @@ OSystem_iOS7::~OSystem_iOS7() {
 	delete _graphicsManager;
 }
 
+#if defined(USE_OPENGL_GAME) || defined(USE_OPENGL_SHADERS)
+Common::Array<uint> OSystem_iOS7::getSupportedAntiAliasingLevels() const {
```

Comment:
Is it possible to share this code across all platforms when `framebufferObjectMultisampleSupported` is in use?

Reply:
I suppose we could but I didn't find an elegant way for it.